jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/menu/ServerTemplateSelectorMenuBuilder.java

```diff
@@ -88,7 +88,7 @@ protected void loadServerTemplates() {
                 widgetView.addServerTemplate(id);
                 view.addServerTemplate(id);
             }
-
+            
```

Comment:
remove trailing whitespace

Reply:
@bxf12315 really minor but can you fix this before merging?

Reply:
@cristianonicolai updated.